.github/workflows/build_and_test_clang_debug.yml

```diff
@@ -55,16 +57,36 @@ jobs:
                               -DKEEP_APOLLO_LOGS=TRUE\
                               -DUSE_FAKE_CLOCK_IN_TIME_SERVICE=TRUE\" "\
               && script -q -e -c "make test"
+        - name: Check ERROR/FATAL logs
+          id: err_report
+          run: |
+            chmod +x "./.github/success_action_if_err_logs_exist.sh"
```

Comment:
@situ-s, I would recommend making success_action_if_err_logs_exist.sh script executable rather than modifying its permissions here and running it.

Reply:
yeah, made the changes and tested them